pkg/apis/cluster/v1alpha1/defaults.go~

```diff
@@ -0,0 +1,20 @@
+/*
```

Comment:
This file shouldn't be in here.... Can you make sure there aren't any other temp files?

Reply:
Gah - sorry, will fix